packages/bot-web-ui/src/components/dashboard/tutorial-tab/common/common-tabs/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import FAQContent from '../../faq-content';
+import GuideContent from '../../guide-content';
+import NoSearchResult from '../no-search-result-found';
+
+export const GuideTab = ({ guideList }) => <GuideContent guide_list={guideList} />;
+
+export const FAQTab = ({ faqList, isMobile }) => <FAQContent faq_list={faqList} hide_header={isMobile} />;
+
+export const SearchTab = ({ filteredList, guideList, faqList, isMobile, search }) =>
+    filteredList.length > 0 ? (
+        <>
+            {guideList.length > 0 && <GuideTab guideList={guideList} />}
+            {faqList.length > 0 && <FAQTab faqList={faqList} isMobile={isMobile} />}
+        </>
+    ) : (
+        <>
+            <NoSearchResult />
+        </>
```

Comment:
do we need to wrap it with fragment? 

```suggestion
            <NoSearchResult />
```